src/loaders/types.js

```diff
@@ -28,5 +29,19 @@ export const fileTypeToLoaderAndSource = {
 };
 
 export function getSourceAndLoaderFromFileType(type) {
-  return fileTypeToLoaderAndSource[type] || [JsonSource, JsonLoader];
+  if (fileTypeToLoaderAndSource[type]) {
+    return fileTypeToLoaderAndSource[type];
+  }
+  if (getPluginFileType(type)) {
```

Comment:
Assign a value to this for its reuse?

Reply:
Done in https://github.com/vitessce/vitessce/pull/1212/commits/8964b41c74fd7782e5c7f7ba901ab8a2966b846b#diff-956d02fec8be803fa226b7c36b72dda712fe2805242fb6dc621540c153317624L35